www/docs/en/dev/guide/hybrid/plugins/index.md

```diff
@@ -224,7 +218,21 @@ To publish a plugin to NPM registry you need to follow steps below:
         $ npm adduser # that is if you don't have an account yet
         $ npm publish /path/to/your/plugin
 
-That is it!
+For more details on npm usage refer to [publishing a npm package](https://docs.npmjs.com/getting-started/publishing-npm-packages) on the NPM documentation site.
+
+## Integrating with Cordova Plugin Search
+To surface the plugin in [Cordova Plugin Search](/plugins/), add the `ecosystem:cordova` keyword to the `package.json` file of your plugin and publish it to npm registry. 
+
+To indicate support for a particular platform add a keyword with the `<platformName>` as "**cordova-<platformName>**" to the list of keywords in package.json. For example, for a plugin that supports android, iOS & Windows, the keywords in package.json should include:
```

Comment:
This is added by plugman too. I think this should say that plugman add this important things. It seems that they have to add them manually.